### ISSUE TITLE
transition_reveal() -- Error: Expected single logical value

### ISSUE THREAD

**Issue by edornshuld** [thomasp85/gganimate#462]
The `transition_reveal()` function results in an error in an Rmd file: `Error: Expected single logical value`

Using the example from the docs:

```
library(ggplot2)
library(gganimate)

anim <- ggplot(airquality, aes(Day, Temp, group = Month)) +
  geom_line() +
  transition_reveal(Day)

anim
```

Session info

```
> sessionInfo(package = NULL)
R version 4.2.1 (2022-06-23 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8    LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                           LC_TIME=English_United States.utf8    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] gganimate_1.0.7 ggplot2_3.3.6  

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.8.3      plyr_1.8.7        pillar_1.7.0      compiler_4.2.1    prettyunits_1.1.1 tools_4.2.1       progress_1.2.2    digest_0.6.29    
 [9] evaluate_0.15     lifecycle_1.0.1   tibble_3.1.7      gtable_0.3.0      pkgconfig_2.0.3   rlang_1.0.2       cli_3.3.0         DBI_1.1.3        
[17] yaml_2.3.5        xfun_0.31         fastmap_1.1.0     withr_2.5.0       dplyr_1.0.9       knitr_1.39        hms_1.1.1         generics_0.1.2   
[25] vctrs_0.4.1       grid_4.2.1        tidyselect_1.1.2  glue_1.6.2        R6_2.5.1          gifski_1.6.6-1    fansi_1.0.3       rmarkdown_2.14.3 
[33] farver_2.1.0      tweenr_1.0.2.9000 purrr_0.3.4       magrittr_2.0.3    scales_1.2.0      ellipsis_0.3.2    htmltools_0.5.2   rsconnect_0.8.26 
[41] assertthat_0.2.1  colorspace_2.0-3  labeling_0.4.2    utf8_1.2.2        stringi_1.7.6     munsell_0.5.0     crayon_1.5.1     
```

**Comment by thomasp85** [comment]
this has been fixed in the dev version of tweenr

**Comment by dafealos** [comment]
I have the same error, How can I fix it? which function of tweenr?

**Comment by pelegr** [comment]
I've also encountered this on a machine running ubuntu 18.04 and the latest R.
The same code is running just fine on my windows based laptop.

Would really appreciate it if someone could point out what package / exact versions are affected,
and how to workaround the issue.

**Update:**
Ended up using remotes to install tweenr 2.0.1 (was on 2.0.0 previously).
The error seems to go away but now it seems to crash in the process with no error given.
I ended up reverting back to tweenr 1.0.2, and the animation successfully renders then.
